.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "kubespray"]
+	path = kubespray
+	url = https://github.com/kubernetes-incubator/kubespray.git
```

Comment:
What's the relationship here to https://github.com/openmicroscopy/kubespray ?

Reply:
https://github.com/openmicroscopy/kubespray allows us to have a tag if we want, currently it's master is out of date.